firstvoices/backend/management/commands/copy_site.py

```diff
@@ -411,32 +399,53 @@ def copy_dictionary_entries_and_return_map(
         entry.site = target_site
         entry.batch_id = ""
         entry.import_job = None
-        entry.save()
+        entry.save(set_modified_date=set_modified_date)
+
+    dictionary_entries = list(DictionaryEntry.objects.filter(site=source_site))
+    # Second pass to set all the relationships
+    for source_entry in dictionary_entries:
+        target_entry_id = dictionary_entry_map[source_entry.id]
+        target_entry = DictionaryEntry.objects.get(id=target_entry_id)
 
+        source_categories = list(source_entry.categories.all())
         updated_categories = [
             category_map[category.id] for category in source_categories
         ]
-        entry.categories.set(updated_categories)
+        target_entry.categories.set(updated_categories)
 
+        source_related_characters = list(source_entry.related_characters.all())
         updated_related_characters = [
             character_map[char.id] for char in source_related_characters
         ]
-        entry.related_characters.set(updated_related_characters)
+        target_entry.related_characters.set(updated_related_characters)
 
+        source_related_entries = list(source_entry.related_dictionary_entries.all())
         updated_related_entries = [
             dictionary_entry_map[source_related_entry.id]
             for source_related_entry in source_related_entries
             if source_related_entry.id in dictionary_entry_map
         ]
-        entry.related_dictionary_entries.set(updated_related_entries)
+        target_entry.related_dictionary_entries.set(updated_related_entries)
 
-        copy_related_media(entry, source_media, audio_map, image_map, video_map)
-        entry.save()
+        source_media = {
+            "audio": list(source_entry.related_audio.values_list("id", flat=True)),
+            "images": list(source_entry.related_images.values_list("id", flat=True)),
```

Comment:
Obviously there aren't any `"documents"`/`related_documents` to copy yet but you may want to add it in here anyway.